networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -370,6 +371,62 @@ def bfs_successors(G, source, depth_limit=None, sort_neighbors=None):
     yield (parent, children)
 
 
+def bfs_layers(G, sources, layer_limit=None):
+    """Returns all the layes of bfs traversal.
+    Parameters
```

Comment:
The documentation building system (Sphinx) needs a blank line after the first "short line" description. The first line is used to describe this function wherever it appears in the docs as a link to this doc_string. So it's nice to have it short and clear (which it is here).  We just need a blank line after it.